.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: "build-test"
+on:
+  push:
+    branches:
+      - net203
```

Comment:
Is there a way to specify the default branch instead of hardcoding one? We've been changing the main branch for each release recently and it'd be great not to have to update it here separately every time.

Reply:
think we can do it with the ```$default-branch``` macro

Reply:
For the record, [here's a sample of syntax](https://github.com/actions/starter-workflows/blob/08ff79c7b930315861d16bed4903ddd90bf7c05c/ci/dotnet-core.yml#L5) if you need one.

Reply:
AIUI, this means run on all pushes to `net203` and all pull requests targeting `net203`. This is effectively the same as "everything", so why not just specify a default `on: [push, pull_request]` or just `on: [pull_request]`? 

This would give you CI results for pushes to an active PR, and simplifies config.

Reply:
@citizenmatt Wouldn't it also run on every push to other branches? I'm not sure if it's a good idea.

Reply:
I could now remember why I always prefer to specify the branch here.

For `on: [push, pull_request]` it will build some of the PRs _twice_, since you push them to this repository, and then create pull requests with them.

If you have three platforms, it becomes messy very quickly:
![image](https://user-images.githubusercontent.com/92793/97083305-40bce400-1639-11eb-875c-de3842c7c9d5.png)

So, let's go with `$default-branch`, and not just `on: [push, pull_request]`.